src/iop/demosaic.c

```diff
@@ -3482,6 +3515,14 @@ void commit_params(struct dt_iop_module_t *self, dt_iop_params_t *params, dt_dev
     d->green_eq = DT_IOP_GREEN_EQ_NO;
     d->color_smoothing = 0;
     d->median_thrs = 0.0f;
+
+    if((pipe->image.flags & DT_IMAGE_RAW) && dt_dev_pixelpipe_uses_downsampled_input(pipe))
+    {
+      // Always enable demosaic for monochrome so we can convert MIPF
+      piece->enabled = 1;
+      // Don't enable opencl as it's not implemented
```

Comment:
Right. I only implemented a CPU code path for that. There's nothing stopping an OpenCL code path but it probably doesn't make sense for such small images.